Google.Api.Gax.Grpc/Rest/RestMethod.cs

```diff
@@ -62,15 +66,7 @@ internal static RestMethod Create(MethodDescriptor method, JsonParser parser)
 
             var pathPattern = HttpRulePathPattern.Parse(pattern, method.InputType);
 
-            Func<IMessage, string> contentFactory = rule.Body switch
-            {
-                "*" => message => message.ToString(),
-                "" => null,
-                string name when method.InputType.FindFieldByName(name) is FieldDescriptor field => message => field.Accessor.GetValue(message).ToString(),
-                _ => throw new ArgumentException($"Method {method.Name} in service {method.Service.Name} has body field {rule.Body} which is not a field in {method.InputType.Name}")
-            };
-
-            return new RestMethod(method, pathPattern, httpMethod, contentFactory, parser);
+            return new RestMethod(method, pathPattern, rule.Body, httpMethod, parser);
```

Comment:
Moving the transcoding switch to later on means we won't find out that the method is misconfigured until we execute it, which I'm not thrilled with. Let's chat about this one.